platform/darwin/src/MGLImageSource.h

```diff
@@ -0,0 +1,59 @@
+#import "MGLSource.h"
+
+#import "MGLFoundation.h"
+#import "MGLTypes.h"
+#import "MGLGeometry.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+MGL_EXPORT
```

Comment:
Similarly, we'll need some documentation here too. See the other source classes' documentation for examples. In particular, we should emphasize that this source is for _georeferenced_ images; the SDK already has several other ways to put images on the map (MGLAnnotationImage, MGLAnnotationView with UIImageView, MGLStyle images).